src/pip/_internal/pep425tags.py

```diff
@@ -115,18 +115,12 @@ def get_abi_tag():
                     lambda: hasattr(sys, 'gettotalrefcount'),
                     warn=(impl == 'cp')):
```

Comment:
I wonder why we only warn for CPython, are these totally sure things on PyPy?

Reply:
For comparison, you may want to look at the newer packaging.tags to see if behavior like this got preserved.

Reply:
The warning is if they’re missing, so I’m guessing it’s because they aren’t known to be required on other implementations (so the warnings would risk being spurious).

Reply:
In `packaging.tags` the behavior did not get preserved, which may be a bug on Windows (filed at pypa/packaging#181).

That seems reasonable.